lib/dpp/DriveStateRepository.js

```diff
@@ -46,6 +49,15 @@ class DriveStateRepository {
     const transaction = this.getDBTransaction('identity');
 
     await this.identityRepository.store(identity, transaction);
+
+    const storeMapPromises = identity.getPublicKeys()
```

Comment:
We need to store the only first key (master key). Also, probably we need separate methods to store/fetch identity? We need the fetch method for the first public key uniqueness validation.

Reply:
@shumkov 

> Also, probably we need separate methods to store/fetch identity? 

Could you, please, explain a bit more where do you want them, why and what those methods do.

>  We need the fetch method for the first public key uniqueness validation.

Could you, please, explain a bit more.

Reply:
We need to store it in a separate method storePublicKeyIdentityId that will be called from the apply state transition function, also we need fetchPublicKeyIdentityId for Identity Create Transition validation function.

Reply:
@shumkov  done